src/game/hud.cpp

```diff
@@ -1810,8 +1832,17 @@ namespace hud
             if((showconsole && showhud) || commandmillis > 0)
             {
                 int numl = consize, numo = consize+conoverflow;
-                loopvj(conlines) if(type ? conlines[j].type >= (confilter && !full ? CON_LO : 0) && conlines[j].type <= CON_HI : conlines[j].type >= (confilter && !full ? CON_LO : 0))
+                //loopvj(conlines) if(type ? conlines[j].type >= (confilter && !full ? CON_LO : 0) && conlines[j].type <= CON_HI : conlines[j].type >= (confilter && !full ? CON_LO : 0))
+                for (int j = consolepos; j < consolepos + min(numo, conlines.length() - consolepos); j++)
                 {
+                    if (type ? conlines[j].type >= (confilter && !full ? CON_LO : 0) && conlines[j].type <= CON_HI : conlines[j].type >= (confilter && !full ? CON_LO : 0))
+                    {
+                    }
+                    else
+                    {
+                        continue;
+                    }
```

Comment:
Avoid empty brackets.

Reply:
will do :D (totally forgot about that line lol)